dockerize/dockerize.py

```diff
@@ -8,26 +13,28 @@
 import shutil
 import subprocess
 import tempfile
+from dockerize.depsolver import DepSolver
```

Comment:
I have had no problem using this from of import in python3 projects.
I have no experience with
```python
from __future__ import absolute_import
```
in python2, but I think it should be fine.